modules/ROOT/partials/configuration/content_css.adoc

```diff
@@ -1,3 +1,4 @@
+[[content_css]]
 == content_css
```

Comment:
```suggestion
== `content_css`
```
This is a bit out of scope, as this was like this in the original markdown so up to you if you fix this.

Reply:
done